### PR TITLE
Gemini Chat App - Python Flask APIs

### PR DESCRIPTION
**Pull request by kinarr**
Use the appropriate Flask endpoints:
- /chat route for the simple chat API
- /stream route for streaming chat API

- A sample .env.example used for environment variables
- README with instructions added to install and run the server

### CODE REVIEW

File: server-python/app.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+from flask import (
```
Comment (kinarr) [inline server-python/app.py:1]: If there's a copyright license header required here LMK

Comment (pcoet) [reply server-python/app.py:1]: Let's use an Apache 2 copyright, e.g. https://github.com/google-gemini/generative-ai-python/blob/main/samples/chat.py#L2-L14

**Review by pcoet — DISMISSED** [review]
LGTM. I might have more suggestions after I try it out with the (forthcoming?) frontend, but looks good so far. Thanks!

File: server-python/app.py (hunk 1 above)
Comment (kinarr) [reply server-python/app.py:14]: Done. I've added it now.

Comment (pcoet) [reply server-python/app.py:14]: Thank you!

**Review by pcoet — APPROVED** [review]

**Pull request merged**